### PR TITLE
GSoC: Add Proposal for KeptnTasks Repository

### PR DESCRIPTION
**Pull request by agardnerIT**

### CODE REVIEW

File: summerofcode/2023.md
Comment (aliok) [inline summerofcode/2023.md:258]: @agardnerIT Can you ask one of the people listed in https://github.com/cncf/foundation/blob/main/project-maintainers.csv#L512 to LGTM this PR? Your name is not in that file and CNCF org admins require the "blessing" of the maintainers from that list.

Comment (thisthat) [reply summerofcode/2023.md:258]: Hey @aliok, I spoke with @agardnerIT and LGTM :)

**Review by thisthat — APPROVED** [review]
LGTM

**Comment by aliok** [comment]
LGTM

**Comment by DavidPHirsch** [comment]
@aliok @nate-double-u looks like this is still stuck.. are there any additional changes needed?

**Comment by aliok** [comment]
IMO it is good. cc @nate-double-u

**Review by nate-double-u — APPROVED** [review]
Thanks for this @agardnerIT, @thisthat & @DavidPHirsch. And thanks for the review help @aliok!

**Pull request merged**
